### PR TITLE
Implementing Holistic Tracking using Python

### LINKED ISSUE #5280: [Machine Learning] Implementing Holistic Tracking using Python
### Topic Suggestion
[Machine Learning] Holistic Tracking using Python

#### Proposed title of article
Your title should be descriptive of the article/tutorial. Be Specific. 

Holistic Tracking using Python

#### Proposed article introduction
*We're looking for the first 2-3 paragraphs of your article that appropriately summarize what your article will be about.*
We will use this writing snippet to help us assess the overall quality before approval so ensure it has been proofread, spellchecked and grammarchecked.

Human pose, face landmarks, and hand tracking in real-time can enable various modern life applications. The MediaPipe library already provides fast and accurate, yet separate, solutions for these tasks. Holistic tracking aims to combine them all in real-time into a semantically consistent end-to-end solution. This is a uniquely difficult problem requiring simultaneous inference of multiple, dependent neural networks.

#### Key takeaways 
What are the 3-5 most important things the reader should understand or be able to do after reading this article?
Use this area to get your ideas down on the bulk of your article or tutorial.

By the end of this tutorial, a reader will be able to understand:
- What is Holistic tracking?
- Components of the Holistic tracking model and how they combine
- Implement holistic tracking in real-time

#### Article quality
How is your article unique? Tell us what makes your approach different from similar articles that have published on the same topic?
Is yours more in-depth? Does it cover additional topics? Do you provide handy tips or anecdotal advice?

I haven't found many tutorials that have implemented the holistic tracking model. This is a unique tutorial.

#### References
Please list links to any published content/research that you intend to use to support/guide this article.
- [Holistic tracking](https://google.github.io/mediapipe/solutions/holistic)

### PR DESCRIPTION
**Pull request by smkinyan**
This closes #5280

## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

`This closes #issue-number`

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by wilkistermumbi** [comment]
@hectorkambow I'd like to review this tutorial.

**Comment by hectorkambow** [comment]
> @hectorkambow I'd like to review this tutorial.

Just noticed the PR - is there any older Pull request that you could review? 
That PR was just submitted less than 24 hours ago - we always try to review oldest to newest in the queue.

**Comment by WanjaMIKE** [comment]
Plagiarism check done

File: content/articles/implementing-holistic-tracking-using-python/index.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+- [Detecting landmarks](#detecting-landmarks)
+- [Wrapping up](#wrapping-up)
+
+### Goal 
+- Setting up the MediaPipe library.
+- Detect poses, facial landmarks, and dual hand poses.
+- Visualize detections on the screen. 
+
+### Introduction
+The [MediaPipe](https://google.github.io/mediapipe/) library provides a wide range of ML solution models. The model that we will use for this tutorial is the [MediaPipe Holistic](https://google.github.io/mediapipe/solutions/holistic).
```
Comment (Bayler) [inline content/articles/implementing-holistic-tracking-using-python/index.md:42]:
```suggestion
The [MediaPipe](https://google.github.io/mediapipe/) library provides a wide range of ML solution models. We will use [MediaPipe Holistic](https://google.github.io/mediapipe/solutions/holistic) model for this tutorial.
```

File: content/articles/implementing-holistic-tracking-using-python/index.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+The MediaPipe holistic model is made up of three different models:
```
Comment (Bayler) [inline content/articles/implementing-holistic-tracking-using-python/index.md:44]:
```suggestion
The MediaPipe holistic model comprises of three different models:
```

File: content/articles/implementing-holistic-tracking-using-python/index.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+- [MediaPipe Pose](https://google.github.io/mediapipe/solutions/pose.html).
+- [MediaPipe Face Mesh](https://google.github.io/mediapipe/solutions/face_mesh.html).
+- [MediaPipe Hands](https://google.github.io/mediapipe/solutions/hands.html).
+
+`MediaPipe Face Mesh` detects the facial landmarks on the face, `MediaPipe Hands` detects all the joints within our hands, and the `MediaPipe Pose` model detects all the poses within the body.
+
+### Installing and importing dependencies
+There are two key dependencies that we will need for this tutorial:
+- MediaPipe. 
+- OpenCV.
```
Comment (Bayler) [inline content/articles/implementing-holistic-tracking-using-python/index.md:54]: Please add external links to these two libraries.

**Comment by Bayler** [comment]
@smkinyan This is an educative tutorial. Please work on the minor revisions before we can move on for final review.

**Comment by smkinyan** [comment]
@Bayler I've finished making the requested changes. Please take a look at it.

**Comment by LinusMuema** [comment]
Hi @smkinyan 

Good job on the article.

cc: @Bayler

**Pull request merged**